src/traceml_ai/launcher/process.py

```
@@ -171,27 +213,41 @@ def terminate_process_group(
     except Exception:
         pass
 
-    try:
-        os.killpg(proc.pid, signal.SIGKILL)
-    except Exception:
+    if _IS_WINDOWS:
+        if not _taskkill_tree(proc.pid, force=True):
+            try:
+                proc.kill()
+            except Exception:
+                pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)